### PR TITLE
New Pug Gridfire Turret and T3 anti missile image

### PR DESCRIPTION
**Pull request by beccabunny**
This PR, based on #4095, introduces a gridfire thumbnail as well as the associated hardpoint and the description. The model is based on the concept of the one made by @10010101001
outfitter
This is the view in the outfitter

### CODE REVIEW

**Comment by beccabunny** [comment]
Any thoughts? @10010101001 do you allow me to credit you in the copiright?

File: data/pug.txt [hunk 1]
```diff
@@ -192,6 +193,7 @@ outfit "Pug Gridfire Turret"
 		"hull damage" 600
 		"blast radius" 40
 		"piercing" .3
+	description "The Gridfire Turret is a very powerful weapon, designed to disable (or as well destroy) even the strongest of the warships."
```
Comment (Pointedstick) [inline data/pug.txt]: Seems unrelated to this patch.

Comment (beccabunny) [reply data/pug.txt]: Are you sure? The other outfits, if they have an image, have got a description as well

Comment (Pointedstick) [reply data/pug.txt]: Argh, sorry, I saw the diff against the previous commit and thought you were removing text that was already there. Ignore me.

Comment (jafdy) [reply data/pug.txt]: It's still quite a bland description, can anyone come up with something better? How about `No one quite knows how the Gridfire Turret works, and the Pug aren't interested explaining it. But it is clearly capable of releasing tremendous energies at any point in space to dice even the strongest of warships.`.
Even if you (probably sensibly) ignore me you should remove the last `...the...` and the `...as well...` for the sake of reading well.

**Comment by Amazinite** [comment]
While you're at it, how about taking a stab at the T3 AMT as well.

File: data/pug.txt (hunk 1 above)
Comment (beccabunny) [reply data/pug.txt]: I'll use yours, wait about 6 hours cuz I don't have time right now

**Comment by beccabunny** [comment]
@Amazinite actually on discord there has been a discussion about making this the AMT and make a different sprite for the gridfire, but i don't laike the idea much since the model i'm working on for the other one will follow the the same concept used on every other turret and, if i have to choose between making the T3 or the Gridfire with a "classic" sprite i would say the T3

**Comment by 10010101001** [comment]
:D

**Comment by Nescio0** [comment]
It would be preferable to design outfit sprites for the Gridfire and the T3AM turret simultaneously, to ensure they're both in the same style.

**Comment by beccabunny** [comment]
[gridfire turret.zip](https://github.com/endless-sky/endless-sky/files/2980070/gridfire.turret.zip)
[T3 PR.zip](https://github.com/endless-sky/endless-sky/files/2980073/T3.PR.zip)
Assets

File: data/pug.txt [hunk 2]
```diff
@@ -192,6 +195,7 @@ outfit "Pug Gridfire Turret"
 		"hull damage" 600
 		"blast radius" 40
 		"piercing" .3
+	description "No one quite knows how the Gridfire Turret works, and the Pug aren't interested explaining it. But it is clearly capable of releasing tremendous energies at any point in space to dice even the strongest of warships."
```
Comment (Lineth) [inline data/pug.txt]: "the Pug aren't interested explaining it" - forgot the 'in' after interested.

'Dice' is also a rather awkward word. Esp. since we don;t really know what the Gridfire actually does to its target (for example, I picture it as tearing ships via spacetime manipulation). Perhaps a more neutral word like 'obliterate'?

Comment (beccabunny) [reply data/pug.txt]: I changed it to disable, better?

**Comment by beccabunny** [comment]
[T3 anti missile.zip](https://github.com/endless-sky/endless-sky/files/2983564/T3.anti.missile.zip)
updated .xcf

File: data/pug.txt [hunk 3]
```diff
@@ -150,6 +151,7 @@ outfit "Tier 3 Anti-Missile"
 		"reload" 6
 		"firing energy" 80
 		"firing heat" 10
+	description "Althought they serve different roles, this powerful anti-missile seems to have a close resemblance to the Pug gridfire turret."
```
Comment (Amazinite) [inline data/pug.txt]:
```suggestion
	description "Although they serve different roles, this powerful anti-missile turret seems to have a close resemblance to the Pug Gridfire Turret."
```

File: data/pug.txt [hunk 4]
```diff
@@ -192,6 +195,7 @@ outfit "Pug Gridfire Turret"
 		"hull damage" 600
 		"blast radius" 40
 		"piercing" .3
+	description "No one quite knows how the Gridfire Turret works, and the Pug aren't interested in explaining it. But it is clearly capable of releasing tremendous energies at any point in space to disable even the strongest of warships."
```
Comment (Amazinite) [inline data/pug.txt]: I think it's fair to just end this after the first sentence.
```suggestion
	description "No one quite knows how the Gridfire Turret works, and the Pug aren't interested in explaining it."
```

File: copyright [hunk 5]
```diff
@@ -508,6 +508,19 @@ Copyright: Lineth (https://github.com/Lineth)
...
+Files:
+images/outfit/pug gridfire turret
+images/hardpoint/pug gridfire turret
+Copyright: Becca Tommaso
+License: CC-BY-SA-3.0
+Comment: Based on 10010101001 (https://github.com/10010101001) concept, derived from works by Michael Zahniser (under the same license) 
+
+Files:
+images/outfit/T3 anti missile
+images/hardpoint/T3 anti missile
```
Comment (Amazinite) [inline copyright]:
```suggestion
 images/hardpoint/T3 anti missile*
```

File: copyright [hunk 6]
```diff
@@ -508,6 +508,19 @@ Copyright: Lineth (https://github.com/Lineth)
...
 
+Files:
+images/outfit/pug gridfire turret
+images/hardpoint/pug gridfire turret
+Copyright: Becca Tommaso
+License: CC-BY-SA-3.0
+Comment: Based on 10010101001 (https://github.com/10010101001) concept, derived from works by Michael Zahniser (under the same license) 
+
+Files:
+images/outfit/T3 anti missile
```
Comment (Amazinite) [inline copyright]:
```suggestion
 images/outfit/T3 anti missile*
```

File: copyright [hunk 7]
```diff
@@ -508,6 +508,19 @@ Copyright: Lineth (https://github.com/Lineth)
 License: CC-BY-SA-4.0
 Comment: Derived from public domain sounds taken from freesound.org.
 
+Files:
+images/outfit/pug gridfire turret
+images/hardpoint/pug gridfire turret
```
Comment (Amazinite) [inline copyright]:
```suggestion
 images/hardpoint/pug gridfire turret*
```

File: copyright [hunk 8]
```diff
@@ -508,6 +508,19 @@ Copyright: Lineth (https://github.com/Lineth)
 License: CC-BY-SA-4.0
 Comment: Derived from public domain sounds taken from freesound.org.
 
+Files:
+images/outfit/pug gridfire turret
```
Comment (Amazinite) [inline copyright]:
```suggestion
 images/outfit/pug gridfire turret*
```

**Comment by Amazinite** [comment]
The high-DPI plugin needs the same copyright changes.

**Comment by beccabunny** [comment]
It should be correct now

File: data/pug.txt [hunk 9]
```diff
@@ -192,6 +195,7 @@ outfit "Pug Gridfire Turret"
 		"hull damage" 600
 		"blast radius" 40
 		"piercing" .3
+	description "No one quite knows how the Gridfire Turret works, and the Pug aren't interested in explaining it."
```
Comment (Bladewood) [inline data/pug.txt]: "Quite knows" is a weird expression. Simply "understands" sounds better to me. Also, I would cut out the "it" at the end.

File: data/pug.txt [hunk 10]
```diff
@@ -136,12 +136,13 @@ effect "pug anti-missile"
 outfit "Tier 3 Anti-Missile"
 	category "Turrets"
 	cost 12000000
-	thumbnail "outfit/unknown"
+	thumbnail "outfit/T3 anti missile"
```
Comment (Bladewood) [inline data/pug.txt]: I'm a bit uncomfortable about how the T is capitalized here.

File: data/pug.txt (hunk 9 above)
Comment (beccabunny) [reply data/pug.txt]: Done both corrections

File: data/pug.txt (hunk 10 above)
Comment (beccabunny) [reply data/pug.txt]: Corrected

**Comment by beccabunny** [comment]
Days ago i asked on Discord and the names Pug Gridfilter Anti-Missile or simply Pug Advanced Anti-Missile were good, but then every old pilot that took the T3 AMT from the Arfecta wouldn't have the image updated

**Comment by jafdy** [comment]
You can still change the image when you depreciate the outfit.

File: copyright [hunk 11]
```diff
@@ -508,6 +508,19 @@ Copyright: Lineth (https://github.com/Lineth)
...
+Copyright: Becca Tommaso
+License: CC-BY-SA-3.0
+Comment: Based on 10010101001 (https://github.com/10010101001) concept, derived from works by Michael Zahniser (under the same license) 
+
+Files:
+ images/outfit/T3?anti?missile*
+ images/hardpoint/T3?anti?missile*
+Copyright: Becca Tommaso
+License: CC-BY-SA-3.0
+Comment: Derived from works by Michael Zahniser (under the same license)
```
Comment (Nescio0) [inline copyright]: Is it? Which works then? Most Pug outfit sprites were made Frederick Goy IV (@comnom).

Comment (beccabunny) [reply copyright]: Right, i corrected it

File: data/pug.txt [hunk 12]
```diff
@@ -150,6 +151,7 @@ outfit "Tier 3 Anti-Missile"
 		"reload" 6
 		"firing energy" 80
 		"firing heat" 10
+	description "Although they serve different roles, this powerful anti-missile turret seems to have a close resemblance to the Pug Gridfire Turret."
```
Comment (Pointedstick) [inline data/pug.txt]: I don't understand this description. The sprites are very different-looking and they serve different purposes (anti-ship vs anti-missile)

Comment (beccabunny) [reply data/pug.txt]: How about now? "Despite they are very different, this powerful anti-missile turret seems to function in a way similar to the Pug Gridfire Turret."

Comment (beccabunny) [reply data/pug.txt]: Changed again, now it is: "Despite being very different in appearance, this powerful anti-missile turret seems to function in a way similar to the Pug Gridfire Turret."

Comment (Pointedstick) [reply data/pug.txt]: I guess I don't understand why any mention of the Gridfire turret needs to be made at all. It's not obvious how they might function in a similar way, nor does this information provide the player with any useful information or function as flavor text.

Comment (MCOfficer) [reply data/pug.txt]: but that's what flavor texts are for, no? we don't really care about the ongoing "war" between lovelace's AM- and sidewinder-team, yet [there it is](https://github.com/endless-sky/endless-sky/blob/master/data/weapons.txt#L730).

Comment (Pointedstick) [reply data/pug.txt]: I'm saying that this isn't actually flavorful enough to count as flavor text. :) It's just an unexplained technical detail that's sort of confusing and context-less.

Comment (beccabunny) [reply data/pug.txt]: Alright, removed. How is it now? "This powerful anti-missile turret is capable of destroying even the strongest of the missiles."

Comment (MCOfficer) [reply data/pug.txt]: *`the strongest of missiles`, unless my english fails me here
